packages/zoltan/src/driver/dr_hg_io.c

```diff
@@ -154,7 +154,7 @@ int read_hypergraph_file(
 {
   /* Local declarations. */
   const char  *yo = "read_hypergraph_file";
-  char   cmesg[256];
+  char   cmesg[FILENAME_MAX+256];
```

Comment:
https://www.gnu.org/software/libc/manual/html_node/Limits-for-Files.html

"Usage Note: Don’t use `FILENAME_MAX` as the size of an array in which to store a file name! You can’t possibly make an array that big! Use dynamic allocation (see Memory Allocation) instead."